matcher/namespaces.go

```diff
@@ -58,7 +58,14 @@ type Namespaces interface {
 
 	// ReverseMatch reverse matches the matching policies for a given id in a given namespace
 	// between [fromNanos, toNanos), taking into account the metric type and aggregation type for the given id.
-	ReverseMatch(namespace, id []byte, fromNanos, toNanos int64, mt metric.Type, at aggregation.Type) rules.MatchResult
+	ReverseMatch(
+		namespace, id []byte,
+		fromNanos, toNanos int64,
+		mt metric.Type,
+		at aggregation.Type,
+		isMultiAggregationTypesAllowed bool,
```

Comment:
should this value be a function call on the `aggregation.TypesOptions`?

Reply:
I thought about it but felt it would be awkward to put it in there. For example, the agg tier uses the types options, but does not need to know which metric types support multiple aggregation types, so it would be kinda strange to use an aggregation types config as part of the agg tier config that doesn't configure the list of metric types supporting multiple aggregation types.